docs/yamls/channel.md

```diff
@@ -12,9 +12,9 @@
 | &rarr; reference  | String that is utilized to define certain tags to the Channel in question, a way for the user to categorize the numerous Channels in the cluster.                        |
 | &rarr;Annotations | Definitions that can describe characteristics of the Channel that later on can be used to process/group the Channels in your cluster.                                    |
 | &rarr; parent     | Defines the Channel context in the cluster through the path of the dApp in which it is stored, for example: `app1.app2` means that the Channel is defined in the `app2`. |
-| &rarr; sha256     | Tags images with their sha256 digest.                                                                                                                                    |
+| &rarr; uuid       | Universally unique identifier for the Channel.                                                                                                                           |
```

Comment:
As this is the YAML definition for channels, I believe there's no need for UUID here as the user will never set it's value